notifier.py

```diff
@@ -196,6 +198,21 @@ def get_user_stars(self, email):
     logging.info('returning %r', feature_ids)
     return feature_ids
 
+  @classmethod
+  def get_feature_starrers(self, feature_id):
+    """Return a list of UserPref objects of all users that starred a feature."""
```

Comment:
If I understand the logic correctly, this comment may be slightly misleading.  The return is the set of users who have starred a feature and also opted in to notifications.

Reply:
OK, since I already landed this CL, I am following-up with a new CL: